test/java/seedu/addressbook/data/person/NameTest.java

```
@@ -5,17 +5,27 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import seedu.addressbook.data.exception.IllegalValueException;
+
 public class NameTest {
     
     private Name name1;
     private Name name2;
     
     @Test
     public void testIsSimilar() {
-        // names are completely equal
-        name1 = new Name("Bob");
-        name2 = new Name("Bob");
-        assertTrue(name1.isSimilar(name2));
+        
+        try {
+            
+            // names are completely equal
+            name1 = new Name("Bob");
+            name2 = new Name("Bob");
+            assertTrue(name1.isSimilar(name2));
+            
+        } catch (IllegalValueException e) {
```

it is better to throw the exception in the test using `throws`